modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1180,6 +1190,34 @@ private ObjectFieldSetting _createObjectFieldSetting(
 		return objectFieldSetting;
 	}
 
+	private void _enableOracleSyntax() {
+		Props props = new PropsImpl();
+
+		String jdbcURL = props.get("jdbc.default.url");
+
+		String className = props.get("jdbc.default.driverClassName");
+
+		if (className.equals(JDBCDriver.class.getName())) {
```

Comment:
@thienquach87 To be able to test the bug scenario the `driverClassName` needs to be `JDBCDriver.class.getName()`, right? Is there any other way to test this without relying on the `driverClassName`?

Reply:
We also can use DBManagerUtil to check this as I implemented in the fixing code.
DB db = DBManagerUtil.getDB();
if (db.getDBType() == DBType.HYPERSONIC)).